### ISSUE TITLE
PYPI stuff

### ISSUE THREAD

**Issue by TomDonoghue** [voytekresearch/neurodsp#36]
I think we can drop the egg-info file on master branch. AFAIK it's not needed - it's actually out of date (it's the info for v0.1 release), so that actually makes it sort of confusing to have hanging around, with the current tagged version (also on PYPI) being 0.2.

Also: we should probably add a description to show up on PYPI:
https://pypi.org/project/neurodsp/

**Comment by srcole** [comment]
PR #38 dropped the files.

As for the PYPI description, we should probably translate the readme into [restructured text](http://www.sphinx-doc.org/en/stable/rest.html). [MNE](https://pypi.python.org/pypi/mne/0.15) has a nice example in the mne.egg-info/PKG-INFO file

**Comment by TomDonoghue** [comment]
Yeah... going for RST is probably the ultimately better option. I didn't bother doing that with FOOOF, and so just copied in a snippet of the README into setup.py, which is also a (somewhat hacky) option.

**Comment by TomDonoghue** [comment]
This is done right? We can close it?
